ansible/era1.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Verify Ansible versions
+  hosts: all
+  tags: always
+  become: false
+  run_once: true
+  gather_facts: false
+  tasks:
+    - local_action: command ./roles.py --check
+      changed_when: false
+
+- name: Configure ERA1 files hosting
+  become: true
+  hosts:
+    - linux-01.ih-eu-mda1.nimbus.sepolia
+  vars:
+    open_ports_default_comment: 'Nginx'
```

Comment:
it would be clearer if those variable are in the `host_vars` file

Reply:
Okay will change.

Reply:
done.